uml/sequence_diagrams/ClockView.txt

```diff
@@ -0,0 +1,46 @@
+@startuml
+participant DeactivatedWatch 
+participant DisplaySelection 
+participant ClockView 
+participant BlueTooth
+participant Battery
+participant DisplayError 
+
+
+opt Activation
+  DeactivatedWatch -> ClockView:  Activate
+end
+opt Clock
+  DisplaySelection --> ClockView : GoTo Clock
+end
+loop event
+alt event == askTimeEvent
+ClockView --> BlueTooth: Ask time
+Activate BlueTooth
+BlueTooth --> ClockView: Time
+Deactivate BlueTooth
+opt Time == TimeError
+  ClockView --> DisplayError : SendError(TimeError)
+  Activate DisplayError
+  DisplayError --> ClockView : DefaultTime (00:00)
+  Deactivate DisplayError
+end
+ClockView --> ClockView: setDisplayTime(Time)
+else event = AskBatteryEvent
+ClockView --> Battery: AskBattery
+Activate Battery
+Battery --> ClockView: Battery
+Deactivate Battery
+opt Battery == BatteryError
+  ClockView --> DisplayError : SendError(BatteryError)
+  Activate DisplayError
+  DisplayError --> ClockView : DefaultBattery (-1%)
+  Deactivate DisplayError
+end
+else event == Selections
+  ClockView --> DisplaySelection : GoTo Selections
+else event == Deactivate
+  ClockView --> DeactivatedWatch : Deactivate

```

Comment:
You have watch deactivation from ClockView. 
I have watch deactivation from Display Selection menu. 
I think this is the requirement:
"The clock shall deactivate when buttons A+B are pressed for more than 3 seconds."
I can update my state diagram if you like, so we only deactivate from Clock view ? 
Perhaps the requirement can also be reworded to mention watch deactivation only from Clock View state ... or something like that

Reply:
No makes sense that deactivation could be from any menu. I will update mines.

Reply:
fixed

Reply:
I have deactivation from Display Selection menu rather than the activated views. 

What we have is different but maybe close enough

Reply:
sorry I didn't get your point initially. As I said in the other PR, I think you should remove the A+B deactivation of Display State Diagram as implicit in any of your diagrams. It is represented anyway in these sequence diagrams.